etomica-modules/src/main/java/etomica/modules/crystalviewer/LatticeEditor.java

```diff
@@ -279,11 +280,7 @@ private void registerValueChangedEvents(DeviceBox[] boxes) {
     }
 
     protected void update() {
-        double[]  boxSize = new double[] { size,size,size };
-
-        box.setBoundary(new BoundaryDeformableLattice
-                                             (currentLattice.getPrimitive(),
-                                              boxSize));
+        box.getBoundary().setBoxSize(Vector.of(size, size, size));

```

Comment:
This won't have the same effect.  `size` here is the number of cells and `currentLattice.getPrimitive()` is the shape of the (not-necessarily-rectangular) cells.  You'll need to do something like

https://github.com/etomica/etomica/blob/master/etomica-core/src/main/java/etomica/space/BoundaryDeformableLattice.java#L38

and then call boundary.setEdgeVector(int,Vector)

or add a convenience method in BoundaryDeformableLattice that takes the primitive and number of cells like the constructor does.